### ISSUE TITLE
FE | Profile | Personal Info page | Add full name and disability rating fields

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#49651]
## Background

The designs have been completed and approved so we can move forward now with adding "Nametag" info to the Personal page. See mocks below:

**Scenario 1:** User does not have rating ([Desktop](https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565/a/eKm18nz) & [Mobile](https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565/a/4ajJxKZ))
**Scenario 2:** User has rating ([Desktop](https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565/a/Gm1KnWY) & [Mobile](https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565/a/eKm18xj)
**Scenario 3:** mockup showing an alert if there's a connection error for disability rating and we can't show that information ([Desktop](https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565/a/v87zKnb) & [Mobile](https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565/a/4ajJxKJ)) 

## Tasks
- [x] Update the Personal info page to include the new "Disability Rating" section near the bottom of the page
- [x] Include the link "Learn more..." : va.gov/disability/view-disability-rating/rating (waiting for confirmation)
- [x] Pact Act link here: va.gov/resources/the-pact-act-and-your-va-benefits/ (waiting for confirmation)
- [x] Conduct Unit tests 
- [x] Show alert for connection error as per mock above
- [x] Run E2E tests covering current code and regression
- [x] Notify PM if this needs to be deployed to Staging behind feature flag
  

## Acceptance Criteria
- [x] Complete all unit testing
- [x] End-to-end tests showing 0 violations
- [x] All axe checks pass
- [x] Notify Product Manager that the change is ready to be validated on Staging
- [x] Incorporate any changes resulting from Staging review & re-test & have re-validated
- [x] Confirm PM has opened a new ticket to deploy changes to Production using feature flag, if needed

## Validation
- Who can validate this ticket? (FE, BE, Design, PM)?
- How can this work be validated?
- What updates need to be made (e.g. product outline, use cases, contact center guide)

## Associated Tickets
- #48990
- analytics ticket?
- staging ticket?

## How to configure this issue
- Include practice area in title -- e.g., [Design], [FE], [BE]
- Include project/initiative name - e.g., Auth Profile: Address Change Messaging
- Add label for practice area (frontend, backend, design)
- Add label for project if applicable
- Assign to a Profile team member
- Associate with an Epic if applicable
- If creating for yourself: estimate work if enough info exists to do so

**Comment by SKasimow** [comment]
Note: Christina is working on whether there needs to be a separate screen with an alert for if disability rating doesn't connect/can't show in general.

**Comment by adamwhitlock1** [comment]
I have a branch up and all parts of the UI are built and functioning. 

I have one question that I have for @Samara-Strauss or @andaleliz about the design. The Mockups show an alert for if the disability rating cannot be accessed/errored. Should we have something similarly for the legal name?

For the current case of the api for full name not succeeding, we have shown a larger alert on the top, and just don't show the main nametag banner at the top at all:

![Screenshot 2023-01-12 at 10.43.43 AM.png](https://images.zenhubusercontent.com/60ddebd504ce2452d0e0fa92/58a4e2f6-8701-4472-aca2-e8464d7eb438)

As you can also see in this screenshot that the legal name section in the personal info page itself is just showing as a label with a blank area for the name. I could add an alert similar to the disability error alert (easiest lift) or I could hide the whole legal name row in the personal info table (harder lift). What do yall think?

**Comment by Samara-Strauss** [comment]
I'll leave this kind of call up to Liz, but I do think we need something in the Legal name field so it's not just blank in the event we can't load it. @andaleliz let me know if this is something you can answer quickly, or if I should add a ticket for you to address this next sprint.

**Comment by andaleliz** [comment]
I think I'm missing something - I don't see what you're talking about where the label is showing with a blank area for the name. The screenshot might be cutoff - I see only the top half of "Connected Apps" in the side nav. Can you post another one, circling what you're talking about? Sorry to be dense :/

**Comment by adamwhitlock1** [comment]
@Samara-Strauss 

We will need designs updated to include an error alert for when Legal Name does not load correctly within the personal information page, as we only have one mocked up for Disability Rating

**Comment by Samara-Strauss** [comment]
Thank you for this reminder, @adamwhitlock1! Ticket created #52064

**Comment by Samara-Strauss** [comment]
Adam -- is it safe to assume this will overflow into next sprint (starts tomorrow)?

**Comment by adamwhitlock1** [comment]
This is now live on staging @Samara-Strauss . Works as expected from my testing.

- Tested with `vets.gov.user+202@gmail.com` and was able to see legal name and disability rating
- Tested with `vets.gov.user+10@gmail.com` as a user with no disability rating
- note: LOA1 users wont see the personal info page at all since they are not identity verified

**Comment by Samara-Strauss** [comment]
Validated with users 10, 80, 202, and an LOA1 user. This looks GREAT! Fantastic work. It's ready to go to prod IMO.

**Comment by Samara-Strauss** [comment]
Just validated this in prod!! It looks great. @adamwhitlock1 feel free to close this when you are ready! Fantastic work!!
